src/Projection/User/UserFinder.php

```diff
@@ -53,6 +53,19 @@ public function findById($userId)
     }
 
     /**
+     * @param string $emailAddress User's email address
+     * @return bool
+     */
+    public function emailAddressExists($emailAddress)
```

Comment:
I would prefer a more generic method here. Something like "getUserWithEmailAddress" could be used in a session service for instance.